src/parse-templates.ts

```diff
@@ -359,37 +325,30 @@ export function parseTemplates(
   return results;
 }
 
-function findImportedNames(
-  template: string,
-  importConfig: StaticImportConfig[]
-): string[] {
-  const importedNames = [];
+function findImportedNames(template: string): Map<string, string> {
+  const importedNames = new Map<string, string>();
   for (const $import of parseStaticImports(template)) {
-    const config = findImportConfigByImportPath(
-      importConfig,
-      $import.moduleName
+    const importIdentifiers = ['default', 'hbs', 'precompileTemplate'];
+    const importModuleNames = [
+      'ember-cli-htmlbars',
+      '@ember/template-compilation',
+      'ember-template-imports',
+      'htmlbars-inline-precompile',
+      'ember-cli-htmlbars-inline-precompile',
+    ];
```

Comment:
I think we still want these to come from the caller like before (`StaticImportConfig[]`)